CMakeLists.txt

```diff
@@ -10,10 +15,15 @@ if(DEFINED ENV{SDKTARGETSYSROOT})
     set(CMAKE_FIND_ROOT_PATH $ENV{SDKTARGETSYSROOT})
 endif()
 
-find_path(MAVLINK_INCLUDE_DIR "mavlink.h"
-    PATH_SUFFIXES "autoquad"
-                  "v1.0/autoquad"
-                  "mavlink/v1.0/autoquad")
+# Generate Mavlink Messages
+set(MAVLINK_INCLUDE_DIR ${CMAKE_CURRENT_BINARY_DIR}/include/mavlink)
+include_directories(${MAVLINK_INCLUDE_DIR} ${MAVLINK_INCLUDE_DIR}/common)
+execute_process(COMMAND ${PYTHON2}
+                ${CMAKE_CURRENT_SOURCE_DIR}/modules/mavlink/pymavlink/tools/mavgen.py
+                ${CMAKE_CURRENT_SOURCE_DIR}/modules/mavlink/message_definitions/v1.0/common.xml
+                --wire-protocol 2.0
```

Comment:
worth noting that having the wire-protocol 2.0 you can still communicate with 1.0